servers/dbConnection.js

```diff
@@ -1,9 +1,8 @@
 const mongoose = require('mongoose');
-// const dbkey = require('../dbConnectionStr.json');
+const dbkey = require('../dbConnectionStr.json');
 
-// const mongodbConnectionStr = process.env.MONGODB_URI;
-const mongodbConnectionStr =
-  'mongodb+srv://plana:Uvc1UKYcdzQJlQsG@cluster0.xwobk.mongodb.net/PlaNA?retryWrites=true&w=majority';
+const mongodbConnectionStr = process.env.MONGODB_URI;
```

Comment:
I think you need to comment out `const mongodbConnectionStr = process.env.MONGODB_URI;`, just leave the one with the json file, not sure if you can assign value to one variable twice.

Reply:
https://github.com/Seneca-PlaNA/body-contouring-clinic/commit/ddf0d2e70c4639ccf27c7a4458f445e201c17c77#diff-ba754de4f166a6110f04e03715725905918bc6202a97e55fd7a7687f4699b7af

The old file before you expose the connection string looks like this, the one with process.env.MONGODB_URI is commented out.